Formula/bzip2.rb

```diff
@@ -29,15 +29,30 @@ def install
     inreplace "Makefile", "$(PREFIX)/man", "$(PREFIX)/share/man"
 
     system "make", "install", "PREFIX=#{prefix}"
+    return if OS.mac?
 
-    if OS.linux?
-      # Install shared libraries
-      system "make", "-f", "Makefile-libbz2_so", "clean"
-      system "make", "-f", "Makefile-libbz2_so"
-      lib.install "libbz2.so.#{version}", "libbz2.so.#{version.major_minor}"
-      lib.install_symlink "libbz2.so.#{version}" => "libbz2.so.#{version.major}"
-      lib.install_symlink "libbz2.so.#{version}" => "libbz2.so"
-    end
+    # Install shared libraries
+    system "make", "-f", "Makefile-libbz2_so", "clean"
+    system "make", "-f", "Makefile-libbz2_so"
+    lib.install "libbz2.so.#{version}", "libbz2.so.#{version.major_minor}"
+    lib.install_symlink "libbz2.so.#{version}" => "libbz2.so.#{version.major}"
+    lib.install_symlink "libbz2.so.#{version}" => "libbz2.so"
+
+    # Create pkgconfig file based on 1.1.x repository.
+    # https://gitlab.com/bzip2/bzip2/-/blob/master/bzip2.pc.in
+    (lib/"pkgconfig/bzip2.pc").write <<~EOS
+      prefix=#{prefix}
+      exec_prefix=${prefix}
+      bindir=${exec_prefix}/bin
+      libdir=${exec_prefix}/lib
+      includedir=${prefix}/include
+
+      Name: bzip2
+      Description: Lossless, block-sorting data compression
+      Version: #{version}
+      Libs: -L${libdir} -lbz2
+      Cflags: -I${includedir}
+    EOS
```

Comment:
Can we not add this as a patch?

Reply:
The 1.1.x repo is separate work from 1.0.x repo (ref: https://sourceware.org/bzip2/downloads.html) and uses different build tools (previously autotools and now cmake/meson) rather than 1.0.x Makefile.

Some Linux distros have been maintaining their own copies of the pkgconfig file, e.g.:
- Alpine - https://git.alpinelinux.org/aports/tree/main/bzip2/bzip2.pc.in
- Arch - https://github.com/archlinux/svntogit-packages/blob/packages/bzip2/trunk/bzip2.pc
- Fedora - https://src.fedoraproject.org/rpms/bzip2/blob/rawhide/f/bzip2.pc

Was mainly testing that this worked. It looks like the rust bindings did dynamically link now (ref: https://github.com/alexcrichton/bzip2-rs/blob/bzip2-sys-0.1.10+1.0.8/bzip2-sys/build.rs#L16-L22)